bower.json

```diff
@@ -17,9 +17,9 @@
     "custom-elements": "webcomponents/custom-elements#remove-mo",
     "es6-promise": "stefanpenner/es6-promise#^4.0.0",
     "html-imports": "webcomponents/html-imports#master",
-    "shadydom": "webcomponents/shadydom#master",
-    "shadycss": "webcomponents/shadycss#master",
-    "template": "webcomponents/template#master",
+    "shadydom": "webcomponents/shadydom#prototype-importNode",
+    "shadycss": "webcomponents/shadycss#polyfill-integration",
+    "template": "webcomponents/template#patch-prototypes",
```

Comment:
But we'll merge these and change them to `master` in `gulp-all-the-things`, then?